ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -275,8 +275,22 @@ case "$CLUSTER_TYPE" in
       echo "VPC Region is ${REGION}"
       echo "PowerVS region is ${POWERVS_REGION}"
       echo "Resource Group is ${RESOURCE_GROUP}"
-      ic resource service-instance-create "${WORKSPACE_NAME}" "${SERVICE_NAME}" "${SERVICE_PLAN_NAME}" "${POWERVS_REGION}" -g "${RESOURCE_GROUP}" --allow-cleanup 2>&1 \
-        | tee /tmp/instance.id
+
+      ##Create a Workspace on a Power Edge Router enabled PowerVS zone
+      # Dev Note: uses a custom loop since we want to redirect errors
+      for i in {1..5}
```

Comment:
Instead of I , we can have a retry_attempt as variable name . otherwise looks good to me .

Reply:
next time... I'll update it ... will have another PR most likely...